aiojobs/_scheduler.py

```diff
@@ -64,21 +64,17 @@ def pending_count(self):
     def closed(self):
         return self._closed
 
-    async def spawn(self, coro):
+    async def spawn(self, coro) -> Job:
```

Comment:
We don't use type annotations in the project yet.
Let's make an another issue for it, convert everything and setup `mypy` checks.
Honestly I doubt if the task could be done until `mypy` support for https://www.python.org/dev/peps/pep-0561/